js/views/file_input_view.js

```diff
@@ -35,7 +35,8 @@
             'click .choose-file': 'open',
             'drop': 'openDropped',
             'dragover': 'showArea',
-            'dragleave': 'hideArea'
+            'dragleave': 'hideArea',
+            'paste': 'onPaste'
```

Comment:
imo, it's better to attach this to the conversation view, capturing paste events in the entire conversation area.